VideoWeb/VideoWeb/ClientApp/src/app/services/hearing-venue-flags.service.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { TestBed } from '@angular/core/testing';
+
+import { HearingVenueFlagsService } from './hearing-venue-flags.service';
+
+describe('HearingVenueFlagsService', () => {
+    let service: HearingVenueFlagsService;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({});
+        service = TestBed.inject(HearingVenueFlagsService);
+    });
+
+    it('getHearingVenueIsScottish returns current value', done => {
+        service.hearingVenueIsScottish$.subscribe(response => {
+            expect(response).toBe(false);
+            done();
+        });
+    });
+
+    it('setHearingVenueIsScottish updates current value', done => {
+        service.setHearingVenueIsScottish(true);
+        service.hearingVenueIsScottish$.subscribe(response => {
+            expect(response).toBe(true);
+            done();
+        });
```

Comment:
You could put this above line 21 just to confirm that a subscription will update after a change.

Reply:
yea cool I will add that assertion in all the tests

Reply:
I just noticed this is in the actual service, if you see the previous test actually tests it being set to false by default, so not adding this change